src/snek5000/reader/pv.py

```diff
@@ -0,0 +1,400 @@
+try:
+    from functools import cached_property
+except ImportError:
+    from cached_property import cached_property
+
+from pathlib import Path
+
+import numpy as np
+import paraview.simple as pv
+from paraview.simple import *  # noqa: F401, F403
```

Comment:
Is there a reason why you need this import ? Anyway, you seem to use `pv.` in this module.

Reply:
From what I remember, the Paraview "server" would need such an import to work. I need to check that again though.

Reply:
By the way, can this module be used from a standard Python? i.e. not from Paraview but by connecting to a Paraview server.

Reply:
Yes, we can use in a "headless" mode without starting the GUI. Not always, but there are some ways where it would work from the standard Python / IPython console:

- The most generic way is to `conda install -c conda-forge paraview`
- In ArchLinux, the [paraview package is built with system python](https://github.com/archlinux/svntogit-community/blob/packages/paraview/trunk/PKGBUILD): `sudo pacman -S paraview; python -c 'import paraview'` works
- The Paraview way of using `pvpython` tool is a bit complicated. And the most common hack is to append the virtual environment site-packages path to `sys.path` or `$PYTHONPATH`. This would be the last resort.